common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -65,23 +75,23 @@
 from django.template.loaders.filesystem import Loader
 engine = Engine(dirs=settings.DEFAULT_TEMPLATE_ENGINE['DIRS'])
 source, template_path = Loader(engine).load_template_source(path)
-%>${source}</%def>
+%>${source | n, decode.utf8}</%def>
 
 <%def name="require_module(module_name, class_name)">
     <script type="text/javascript">
         (function (require) {
             % if settings.REQUIRE_DEBUG:
-                require(['${module_name}'], function (${class_name}) {
-                    ${caller.body()}
+                require(['${module_name | n, js_escaped_string}'], function (${class_name | n, decode.utf8}) {
```

Comment:
- The filter `js_escaped_string` can only be used on an expression enclosed in quotes.
- The expression with `caller.body()` is not escaped here because it will have been properly JavaScript escaped before getting to this point, in whatever template this def is used.  The linter lints for that.
- I had discussed with Ned whether $class_name needed its own filter, like `safe_identifier` (or whatever the name might be), but we determined that: 1) This is an outlier case to have a class name in an expression. 2) It seems highly unlikely that we would ever get the classname from user supplied data. and 3) We need(ed) to draw a line somewhere.  You are correct, as written, it would be unsafe if someone did allow a user to supply this class name.